scripts/dataset_processing/tts/thorsten_neutral/get_data.py

```diff
@@ -0,0 +1,263 @@
+    )
+    parser.add_argument("--data-root", required=True, type=Path, help="where the resulting dataset will reside.")
+    parser.add_argument("--manifests-root", required=True, type=Path, help="where the manifests files will reside.")
+    parser.add_argument("--data-version", default="22_10", choices=["21_02", "22_10"], type=str)
+    parser.add_argument("--min-duration", default=0.1, type=float)
+    parser.add_argument("--max-duration", default=float('inf'), type=float)
+    parser.add_argument("--val-size", default=100, type=int)
+    parser.add_argument("--test-size", default=100, type=int)
+    parser.add_argument(
+        "--num-workers",
+        default=-1,
+        type=int,
+        help="Specify the max number of concurrently Python workers processes. "
+        "If -1 all CPUs are used. If 1 no parallel computing is used.",
+    )
+    parser.add_argument(
+        "--normalize-text",
+        default=False,
+        action='store_true',
+        help="Normalize original text and add a new entry 'normalized_text' to .json file if True.",
+    )
+    parser.add_argument(
+        "--seed-for-ds-split",
+        default=100,
+        type=float,
+        help="Seed for deterministic split of train/dev/test, NVIDIA's default is 100.",
+    )
+    args = parser.parse_args()
+    return args
+
+
+def __maybe_download_file(source_url, destination_path):
+    if not destination_path.exists():
+        logging.info(f"Downloading data: {source_url} --> {destination_path}")
+        tmp_file_path = destination_path.with_suffix(".tmp")
+        urllib.request.urlretrieve(source_url, filename=tmp_file_path)
+        tmp_file_path.rename(destination_path)
+    else:
+        logging.info(f"Skipped downloading data because it exists: {destination_path}")
+
+
+def __extract_file(filepath, data_dir):
+    logging.info(f"Unzipping data: {filepath} --> {data_dir}")
+    shutil.unpack_archive(filepath, data_dir)
+    logging.info(f"Unzipping data is complete: {filepath}.")
+
+
+def __save_json(json_file, dict_list):
+    logging.info(f"Saving JSON split to {json_file}.")
+    with open(json_file, "w") as f:
+        for d in dict_list:
+            f.write(json.dumps(d) + "\n")
+
+
+def __text_normalization(json_file, num_workers=-1):
+    text_normalizer_call_kwargs = {
+        "punct_pre_process": True,
+        "punct_post_process": True,
+    }
+    text_normalizer = Normalizer(
+        lang="de", input_case="cased", overwrite_cache=True, cache_dir=str(json_file.parent / "cache_dir"),
+    )
+
+    def normalizer_call(x):
+        return text_normalizer.normalize(x, **text_normalizer_call_kwargs)
+
+    def add_normalized_text(line_dict):
+        normalized_text = normalizer_call(line_dict["text"])
+        line_dict.update({"normalized_text": normalized_text})
+        return line_dict
+
+    logging.info(f"Normalizing text for {json_file}.")
+    with open(json_file, 'r', encoding='utf-8') as fjson:
+        lines = fjson.readlines()
```

Comment:
nice! are you suggesting adding this instruction in the `logging.info` above for future changes?

Reply:
I think it's fine without the logging.info to avoid confusion and since we don't have the warning anywhere else. It was mostly FYI that for new datasets the processing should be easier. 
When we do the switch, we have to go through all normalizer imports and check that nothing is broken. Is this script included in the QA plan?

Reply:
I see. I don't think any data process scripts are involved with any QA, but we have QA for tutorials. We could plug these data process scripts into tutorials so that they can be well QAed along with tutorials.